### PR TITLE
pages/*, style-guide: update contents, fix Markdown

### PR DESCRIPTION
**Pull request by kbdharun**
## Changes

- Update `arg` to `argument`.
- Update `package_name` to `package` (in applicable places).
- Update `kubectl-run` page.
- Fix Markdown (code blocks, lists, etc) and spacing in style guide pages. [Note: I have intentionally avoided the markdown-lint list warning in the Chinese section on `style-guide.md`].

> I have been working on this PR for a while so sorry for the multiple commits, I would suggest "Squash & Merging" this PR with a clean commit message once approved.

### CODE REVIEW

File: pages/linux/pkgfile.md [hunk 1]
```diff
@@ -14,11 +14,11 @@
 
 - List all files provided by a package:
 
-`pkgfile --list {{package_name}}`
+`pkgfile --list {{package}}`
 
 - List only files provided by a package located within the `bin` or `sbin` directory:
```
Comment (acuteenvy) [inline pages/linux/pkgfile.md:19]:
```suggestion
- List only files provided by a package located within the `/bin` or `/sbin` directory:
```

File: pages/common/pip3.md [hunk 2]
```diff
@@ -3,25 +3,21 @@
 > Python package manager.
 > More information: <https://pip.pypa.io>.
 
-- Find available packages:
-
-`pip3 search {{package_name}}`
-
```
Comment (pixelcmtd) [inline pages/common/pip3.md:9]: What's your rationale for that?

Comment (kbdharun) [reply pages/common/pip3.md:9]: @acuteenvy informed in a previous review comment that this behavior has been removed from pip.

Comment (acuteenvy) [reply pages/common/pip3.md:9]: https://github.com/tldr-pages/tldr/pull/10492#discussion_r1289021279

Comment (pixelcmtd) [reply pages/common/pip3.md:9]: What a weird thing of PyPA to do

File: pages/linux/pkgfile.md (hunk 1 above)
Comment (pixelcmtd) [reply pages/linux/pkgfile.md]: But doesn't this refer to `bin` and `sbin` inside the package?

**Review by pixelcmtd — COMMENTED** [review]
77/83 files viewed

File: pages/linux/rpm.md [hunk 3]
```diff
@@ -14,7 +14,7 @@
 
 - Forcibly install a package regardless of currently installed versions:
 
-`rpm --upgrade {{package_name.rpm}} --force`
+`rpm --upgrade {{package.rpm}} --force`
```
[the remark this answers is not in the document]
Comment (kbdharun) [reply pages/linux/rpm.md:17]: Yeah, I will apply the first suggestion.

File: pages/linux/pkgfile.md (hunk 1 above)
Comment (kbdharun) [reply pages/linux/pkgfile.md]: Probably yes (https://man.archlinux.org/man/extra/pkgfile/pkgfile.1.en), will wait for confirmation from @acuteenvy.

**Review by blueskyson — APPROVED** [review]

**Comment by blueskyson** [comment]
I noticed that some placeholders are `query`. Maybe you can mention that the user can simply use the package name as a query.

File: pages/linux/pkgfile.md (hunk 1 above)
Comment (acuteenvy) [reply pages/linux/pkgfile.md]: > But doesn't this refer to `bin` and `sbin` inside the package?

It does, but the package is unpacked to the same directory on the system so it doesn't really matter.

---

Also, both `/bin` and `/sbin` are symlinks to `/usr/bin` on Arch, and packages always use `/usr/bin`:

https://wiki.archlinux.org/title/Arch_package_guidelines#Directories
> - Packages should not contain any of the following directories:
> /bin
> /sbin
> ...

I'll make another suggestion.

File: pages/linux/pkgfile.md [hunk 4]
```diff
@@ -14,11 +14,11 @@
 
 - List all files provided by a package:
 
-`pkgfile --list {{package_name}}`
+`pkgfile --list {{package}}`
 
-- List only files provided by a package located within the `bin` or `sbin` directory:
+- List only files provided by a package located within the `/bin` or `/sbin` directory:
```
Comment (acuteenvy) [inline pages/linux/pkgfile.md:19]:
```suggestion
- List executables provided by a package:
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**